sdk/table/azure-table/azure/table/_table_service_client.py

```diff
@@ -0,0 +1,284 @@
+
+        :return: Dictionary of Service Stats
+        :rtype:dict[str, object]
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        try:
+            timeout = kwargs.pop('timeout', None)
+            stats = self._client.service.get_statistics(  # type: ignore
+                timeout=timeout, use_location=LocationMode.SECONDARY, **kwargs)
+            return service_stats_deserialize(stats)
+        except HttpResponseError as error:
+            process_table_error(error)
+
+    @distributed_trace
+    def get_service_properties(self, **kwargs):
+        # type: (...) -> dict[str,Any]
+        """Gets the properties of an account's Table service,
+        including properties for Analytics and CORS (Cross-Origin Resource Sharing) rules.
+
+        :return: Dictionary of service properties
+        :rtype:dict[str, Any]
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        timeout = kwargs.pop('timeout', None)
+        try:
+            service_props = self._client.service.get_properties(timeout=timeout, **kwargs)  # type: ignore
+            return service_properties_deserialize(service_props)
+        except HttpResponseError as error:
+            process_table_error(error)
+
+    @distributed_trace
+    def set_service_properties(
+            self,
+            analytics_logging=None,  # type: Optional[TableAnalyticsLogging]
+            hour_metrics=None,  # type: Optional[Metrics]
+            minute_metrics=None,  # type: Optional[Metrics]
+            cors=None,  # type: Optional[CorsRule]
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        """Sets properties for an account's Table service endpoint,
+        including properties for Analytics and CORS (Cross-Origin Resource Sharing) rules.
+
+       :param analytics_logging: Properties for analytics
+       :type analytics_logging: ~azure.table.TableAnalyticsLogging
+       :param hour_metrics: Hour level metrics
+       :type hour_metrics: ~azure.table.Metrics
+       :param minute_metrics: Minute level metrics
+       :type minute_metrics: ~azure.table.Metrics
+       :param cors: Cross-origin resource sharing rules
+       :type cors: ~azure.table.CorsRule
+       :return: None
+       :rtype: None
+       :raises: ~azure.core.exceptions.HttpResponseError
+       """
+        props = TableServiceProperties(
+            logging=analytics_logging,
+            hour_metrics=hour_metrics,
+            minute_metrics=minute_metrics,
+            cors=cors
+        )
+        try:
+            return self._client.service.set_properties(props, **kwargs)  # type: ignore
+        except HttpResponseError as error:
+            process_table_error(error)
+
+    @distributed_trace
+    def create_table(
+            self,
+            table_name,  # type: str
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> TableClient
+        """Creates a new table under the given account.
```

Comment:
nit: probably better to not say "given account" in this context since it seems like you're giving an account to `create_table`. Can say using the current client's storage account